### DISCUSSION TITLE
Dynamic variables in custom SCA scan

### DISCUSSION THREAD

**Discussion by bekefi-laszlo** [wazuh/wazuh#16391] [Q&A]
Hi,

Is it possible to define a variable in a custom SCA policy in a dynamiv way?
I mean that instead of: $vim_path: /usr/bin/vim I could use the 'which vim' command, to get to know the path.

Thanks,
Laszlo

**Comment by mdiego92** [comment]
Hello @bekefi-laszlo,

Yes, if you previously define the variable you should be able to get the output of that command.

e.g.:

```
variables:
  $vivariable: which vi
```
```
    rules:
      - 'c:$vivariable -> r:/usr/bin/vi'
```



Here you have another case of use from the [cis_apache_24](https://github.com/wazuh/wazuh/blob/master/ruleset/sca/applications/cis_apache_24.yml) policy:

`  $enabled-modules: httpd -M
`

Documentation for reference: https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/creating-custom-policies.html#sca-policy-file-variables-section


Please let me know if it worked for you!
Regards.
